extras/build_hashes.py

```diff
@@ -11,5 +14,11 @@
             for filename in os.listdir(entry_path):
                 path = os.path.join(entry_path, filename)
                 hash = hashlib.sha256(open(path).read()).hexdigest()
-                print path, hash
-                output.write('{} {}\n'.format(hash, os.path.basename(path)))
+                if hash not in hashes:
+                    print path, hash
+                    output.write('{} {}\n'.format(hash, os.path.basename(path)))
+                    hashes[hash] = path
+                else:
+                    print "Error: {} and {} have the same hash ({})...Exiting.".format(path, hashes[hash], hash)
```

Comment:
Does this not normally happen? How do you resolve a case where they hash the same or do we never include multiple sources for the same file? ie jQuery from GitHub and from CDN JS?